src/lv_core/lv_obj.c

```diff
@@ -2950,6 +2964,18 @@ bool lv_obj_get_click(const lv_obj_t * obj)
     return obj->click == 0 ? false : true;
 }
 
+/**
+ * Get the passing of event of an object if it is disabled
+ * @param obj pointer to an object
+ * @param true: the object does not pass events if it is disabled
+ */
+bool lv_obj_get_disabled_noevent(lv_obj_t * obj, bool en)
```

Comment:
I think this is a copy-paste issue.

Reply:
Yes, it is. I have deleted the "bool en" in my fork, but I don't know how to update this pull request

Reply:
The name does not suit me very much, but I could not find anything better. Feel free to change it for whatever you like more

Reply:
Pushing to your branch updates it automatically. I will fix the header and change the name.

Reply:
Mine really is a long name; maybe we should wait for @kisvegabor to weigh in.

Reply:
OK, I have already corrected it in my branch